README.md

```diff
@@ -1,2 +1,13 @@
 # HW01
-Edit the readme
+## Random edit
+r <- 8
+r + r
+
+## About me
+
+* I am currently living in Oregon.
+* I will be moving to Chicago in September.
+* I just finished reading *An American Marriage*; I recommend it.
+* I really enjoyed watching ATLA last month; it was **really good**.
+* Here is a link to [github](www.github.com)
+* Here is a meme I enjoy ![meme](Image-1.jpg)
```

Comment:
Love the meme, it very much describes my current feelings towards everything. However, the image didn't show up for my in the preview of the code. For the image in my assignment, I used this code : <img src="Photo of Me! 2.0.jpg" width="500" height="400" /> From the website I got this code from, it said it was html syntax that seems to work in Markdown. I had to use this because it allowed me to manually alter the size of the image, since mine was too big by default. I hope that was useful!

Reply:
Oh no! Yes, I was having issues with my image insertion. In my latest version I thought I saw the image in preview, but now I can't anymore. I will have to try your suggestion; thank you!